platformio.ini

```diff
@@ -6,8 +6,10 @@ build_flags =
   -D CONFIG_ARDUHAL_LOG_COLORS
   -D CORE_DEBUG_LEVEL=ARDUHAL_LOG_LEVEL_DEBUG
   -D MYCILA_JSON_SUPPORT
+;  -D MYCILA_NEOPIXEL_SUPPORT
 lib_deps = 
   bblanchon/ArduinoJson @ 7.2.0
+  adafruit/Adafruit NeoPixel @ 1.12.3
```

Comment:
I've reworked the CI and pio file in order to be able to add an isolated section for neopixel (like it is done with json) in order to test it better in CI

Reply:
Not sure if I need to do something on this one?

Reply:
I updated the CI part ;-)